<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Timestamp2Time</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="5">
     <widget class="QLabel" name="bjTimeLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>Beijing time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="fromBjConvertButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>convert</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="fromBjTimeEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="fromTsEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="fromTsLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>timestamp</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="curTimestampLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>current timestamp</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="fromBjTimeLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>Beijing time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="tsToTimeConvertButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>convert</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="bjTimeLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="toTimestampLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>timestamp</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="toTimeStampEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="curTimeStampEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="startStopButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
